packages/react-dom/src/events/__tests__/DOMPluginEventSystem-test.internal.js

```diff
@@ -1007,7 +1007,7 @@ describe('DOMPluginEventSystem', () => {
           const divElement = divRef.current;
           divElement.focus();
           expect(onFocus).toHaveBeenCalledTimes(1);
-          expect(onFocusCapture).toHaveBeenCalledTimes(1);
```

Comment:
This expectation didn't make sense; even if it were stopped in the bubble phase React still should have captured it (twice — in both cases where the handler is used). I'm changing this to `3`, like in the original click test.